### PR TITLE
Added dimension for new language metadata

### PR DESCRIPTION
**Pull request by BJenning**

### CODE REVIEW

File: cmslite_metadata.view.lkml [hunk 1]
```diff
@@ -12,4 +12,9 @@ view: cmslite_metadata {
   dimension: title {}
   dimension: hr_url {}
   dimension: language {}
+  dimension: page_language {
+    type: string
+    sql: ${TABLE}.language_name ;;
+    description: "The language specified in the page settings"
```
Comment (mark-walle) [inline cmslite_metadata.view.lkml]: Can you make this "The language specified in the CMS Lite page settings tab"?

Comment (BJenning) [reply cmslite_metadata.view.lkml]: Ive updated the description. Language code was added by the developer to the script, but it wasn't requested by us. I added a dimension for it as per your suggestion however.

File: cmslite_metadata.view.lkml [hunk 2]
```diff
@@ -12,4 +12,14 @@ view: cmslite_metadata {
...
   dimension: hr_url {}
   dimension: language {}
+  dimension: page_language {
+    type: string
+    sql: ${TABLE}.language_name ;;
+    description: "The language specified in the CMS Lite page settings tab"
+  }
+  dimension: language_code {
+    type: string
+    sql: ${TABLE}.language_name ;;
```
Comment (mark-walle) [inline cmslite_metadata.view.lkml]: Is the language code actually specified, or is it just a key associated to the language specified?

Comment (BJenning) [reply cmslite_metadata.view.lkml]: It isnt specified in page settings in cmslite, so it appears to just be a key associated to the language value.

**Pull request merged**